DigitalLearningSolutions.Data.Tests/Helpers/CustomPromptsTestHelper.cs

```diff
@@ -1,96 +1,96 @@
-﻿namespace DigitalLearningSolutions.Data.Tests.Helpers
-{
-    using System.Collections.Generic;
-    using DigitalLearningSolutions.Data.Models.CustomPrompts;
-
-    public static class CustomPromptsTestHelper
-    {
-        public static CentreCustomPrompts GetDefaultCentreCustomPrompts
+﻿namespace DigitalLearningSolutions.Data.Tests.Helpers

```

Comment:
Could we change this back to CRLF + other affected files? Resharper's Cleanup Code should do the magic.. :)